Feat(silent Alert): Updates in alert condition threshold setup (#20395)
newrelic/docs-website: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx

        You need to set the priority level for both anomaly and static thresholds.  See the section above for more details.
      </Collapser>

            <Collapser
        id="disable-health-status-reporting"
        title="Health status reporting"
      >

>         In New Relic, entities are associated with [color-coded health statuses](/docs/alerts/create-alert/examples/view-entity-health-status-find-entities-without-alert-conditions/#colors). You can view the current state of these entities from their respective indexes, maps and various other places. When an alert condition is associated to an entity, the entity's health status is determined by the alert condition. If the alert triggers an incident, the entity's health status changes based on the incident's severity level.

Reviewer: This is something we should avoid - "various other places". I would list the most important ones and leave the rest for the user to learn by observation or doing.

        If you want the alert condition to not affect the health status of the associated entity, enable the <DNT>**Do not report system health status**</DNT> toggle. This is useful when you want to monitor an entity without affecting its overall health status.

        <Callout variant="important">

>           When you are creating a cross-account alert condition, the <DNT>**Do not report system health status**</DNT> toggle is disabled by default. To prevent the cross-account alert condition from determining the health status of the associated entity, you can enable the toggle.

Reviewer: ```suggestion
          When you create a cross-account alert condition, the <DNT>**Do not report system health status**</DNT> toggle is disabled by default. To prevent the cross-account alert condition from determining the health status of the associated entity, enable it.
```

Author: In the process of creating a cross-account alert condition, the user needs to enable this toggle, if needed. So, I am keeping it as "When you are creating a cross-account alert condition...".
